cl/alerts/tasks.py

```diff
@@ -69,6 +71,13 @@ def make_alert_messages(
     return messages
 
 
+def make_docket_alert_webhook_payload(docket: Docket) -> Dict:
+    return {
```

Comment:
Biggest question for you, what do we want in this payload? We are also providing a public S3 link for the document, correct?

Reply:
I think the most logical approach to this is to use our existing API format to send something like this:

https://www.courtlistener.com/api/rest/v3/docket-entries/?docket=18507805&recap_documents__document_number=1

If we use this format, it makes our API consistent both as webhooks and as GET requests, which would be really nice. I think it shouldn't be too hard either. We have code for serializing objects into API responses here: 

https://github.com/freelawproject/courtlistener/blob/c029929f6285833f9c22ed4a95211e3d01f712b2/cl/corpus_importer/tasks.py#L188-L193

The general idea is just this though:

```py3
d = Docket.objects.get(pk=xyz)
renderer = JSONRenderer()
json_str = renderer.render(
    IADocketSerializer(d).data,
    accepted_media_type="application/json; indent=2",
).decode()
return d, json_str
```

I think you could apply that same pattern to each of the items in `new_ds`, and have a list of nice docket entry objects that you could use. You'd just need the serializer for docket entries instead of the one for dockets, which is here: `cl.search.api_serializers.DocketEntrySerializer`. Note that the query that normally builds that object has some query optimizations is in `cl.search.api_views.DocketEntryViewSet`. Maybe borrow those as well? 

So I think that will get you JSON for the docket entries with nested information for the RECAPDocuments and Docket. What's missing then, is some metadata about the hook itself. I'd propose:

1. The date created in a `date_created` field
2. The version of the webhook (start with it matching the API, so 3.13).
3. `event_type`: should match with the values in `cl.api.models.WebhookEvent`

So, that's a lot. I think where that leaves us is:

1. Use the serializers to serialize the docket entries you already have in the function.
2. Make a new json object that contains something like:

    ```json
    {
      "webhook": {
        "event_type": "2",  # Alert triggered
        "version": "3.13",
        "date_created": "ISO-8601-date",
      },
      "results": {
        ....the serialized docket entries...
      }
    }
    ```

    I'm not convinced that putting version, date, and type into the webhook key is right, but it seems pretty good.

Does this feel like a good place for you to begin?